Based on patches provided by John Dey from Fred Hutch:
https://github.com/FredHutch/easybuild-life-sciences/blob/main/fh_easyconfigs/c/CellProfiler/CellProfiler-4.2.4-foss-2021b.eb
Uploaded: J. Sassmannshausen (Imperial College London/UK)
diff -ruN centrosome-1.2.1.orig/setup.py centrosome-1.2.1/setup.py
--- centrosome-1.2.1.orig/setup.py	2022-07-29 07:15:00.000000000 -0700
+++ centrosome-1.2.1/setup.py	2022-09-27 15:40:27.796375000 -0700
@@ -118,7 +118,7 @@
     },
     install_requires=[
         "deprecation",
-        "matplotlib==3.1.3",
+        "matplotlib>=3.1.3",
         "numpy>=1.18.2",
         "pillow>=7.1.0",
         "scikit-image>=0.17.2",
